confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/AbstractMacroConverter.java

```diff
@@ -0,0 +1,153 @@
+        Collection<String> unhandledParameters = confluenceParameters.getUnhandledParameters();
+
+        for (String p : unhandledParameters) {
+            if (!parametersWithUnhandledValues.contains(p)) {
+                logger.info(UNHANDLED_PARAMETER_MARKER, "Unhandled parameter [{}] (with value [{}]) in macro [{}]",
+                    p, confluenceParameters.get(p), confluenceId);
+            }
+        }
+
+        for (String p : parametersWithUnhandledValues) {
+            logger.info(UNHANDLED_PARAMETER_VALUE_MARKER, "Unhandled value [{}] for parameter [{}] in macro [{}]",
+                confluenceParameters.get(p), p, confluenceId);
+        }
+    }
+
+    @Override
+    public String toXWikiId(String confluenceId, Map<String, String> confluenceParameters, String confluenceContent,
+        boolean inline)
+    {
+        return confluenceId;
+    }
+
+    /**
+     * Convert the macro content.
+     * @param confluenceId the name of the macro in Confluence
+     * @param parameters the parameters of the macro
+     * @param confluenceContent the content in Confluence
+     * @return the content in XWiki
+     */
+    protected String toXWikiContent(String confluenceId, Map<String, String> parameters, String confluenceContent)
+    {
+        return confluenceContent;
+    }
+
+    /**
+     * Mark a Confluence parameter as (un)handled.
+     * @param confluenceParameters the parameters in Confluence, as passed to toXWikiParameters
+     * @param name the name of the parameter to mark as handled
+     * @param handled whether the parameter is to be marked as handled
+     */
+    protected void markHandledParameter(Map<String, String> confluenceParameters, String name, boolean handled)
+    {
+        if (confluenceParameters instanceof TracedMap) {
+            TracedMap<String, String> tracedParameters = (TracedMap<String, String>) confluenceParameters;
+            if (handled) {
+                tracedParameters.markAsUsed(name);
+            } else {
+                tracedParameters.markAsUnused(name);
+            }
+        } else {
+            warnCantMark();
+        }
+    }
+
+    /**
+     * Mark a Confluence parameter value as unhandled.
+     * @param confluenceParameters the parameters in Confluence, as passed to toXWikiParameters
+     * @param parameterName the name of the parameter of which the value is unhandled.
+     */
+    protected void markUnhandledParameterValue(Map<String, String> confluenceParameters, String parameterName)
+    {
+        if (confluenceParameters instanceof TracedMap) {
+            TracedMap<String, String> tracedParameters = (TracedMap<String, String>) confluenceParameters;
+            tracedParameters.markAsUnhandledValue(parameterName);
+        } else {
+            warnCantMark();
+        }
+    }
+
+    private void warnCantMark()
+    {
+        logger.error(
```

Comment:
The method name suggest the intent is `.warn`, but maybe the method name is wrong ?